myst_nbs/case_studies/item_response_nba.myst.md

```diff
@@ -6,11 +6,20 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
+(item_response_nba)=
+# NBA Foul Analysis with Item Response Theory
+
+:::{post} Apr 17, 2022
+:tags: hierarchical model, case study, generalized linear model
+:category: intermediate, tutorial
+:author:  Michael Osthege, Lorenzo Toniazzi
```

Comment:
People who only re-execute the notebook should not be listed as authors. Here you (or you and Austin) should be the authors.

Also related but not here, I think there was an awknowledgement thanking Eric that I can't find anymore, if that was to comply with the authorship guidance, there is no need to remove awknowledgements.

Reply:
Ok, but should I remove Osthege in the re-execution line from the ##Authors section?

Reply:
No, the authors section should have everybody but the metadata (and therefore citation) should exclude people who only re-executed the notebook. The reason for this is to address the concerns when adding the citation recomendation that people might game the system by reruning notebooks that do not need to be re-executed in order to appear there.